tests/Microsoft.Graph.DotnetCore.Test/Requests/Functional/ExcelTests.cs

```diff
@@ -36,11 +36,13 @@ public async Async.Task OneDriveCreateDeleteExcelWorkbook()
             }
         }
 
+        [Theory(Skip = "No CI set up for functional tests")]
+        [InlineData("_excelTestResource.xlsx")]
```

Comment:
Does this file need to be checked in?

Reply:
We create it with OneDriveCreateTestFile.

We rarely run these ad hoc tests. Goal is to clean up these warnings associated with these tests.